### PR TITLE
PMM-10270 PMM-12191 Feature build

### PR DESCRIPTION
**Pull request by artemgavrilov**
https://jira.percona.com/browse/PMM-10270

- [ ] https://github.com/percona/pmm/pull/2202
- [ ] https://github.com/percona-platform/grafana/pull/672

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3295-cf9e54d
client docker - perconalab/pmm-client-fb:PR-3295-cf9e54d
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3295-cf9e54d.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3295-cf9e54d&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3295-cf9e54d.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3295-0c426bd
client docker - perconalab/pmm-client-fb:PR-3295-0c426bd
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3295-0c426bd.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3295-0c426bd&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3295-0c426bd.tar.gz

File: ci.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+deps:
+  # COMMON
+  - name: pmm
+    branch: PMM-10270-restore-mongo-backup-on-new-service
```
Comment (artemgavrilov) [inline ci.yml]:
```suggestion
    branch: PMM-10270-restore-mongo-backup-on-new-service
 - name: grafana
   branch: 12191-compatible-target-services
    
```

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3295-0ebbe8b
client docker - perconalab/pmm-client-fb:PR-3295-0ebbe8b
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3295-0ebbe8b.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3295-0ebbe8b&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3295-0ebbe8b.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/13855/

**Comment by artemgavrilov** [comment]
Recreated: https://github.com/Percona-Lab/pmm-submodules/pull/3334

**Pull request closed without merging**